frontend/src/core/components/InfoCard/InfoCard.js

```diff
@@ -3,39 +3,23 @@ import PropTypes from "prop-types";
 import "./InfoCard.scss";
 
 class InfoCard extends Component {
-  constructor(props, context) {
-    super(props, context);
-
-    this.renderChildren = this.renderChildren.bind(this);
-  }
-
-  renderChildren() {
-    const {children} = this.props;
-
-    if (children !== undefined) {
-      return children;
-    } else {
-      return <br></br>;
-    }
-  }
-
   render() {
-    const {title, subtitle, className} = this.props;
+    const {title, subtitle, className, children} = this.props;
 
     return (
       <div className={"p-badge " + className}>
         <div className="p-badge-body">
           <h2>{title}</h2>
           <p>{subtitle}</p>
-          {this.renderChildren()}
+          {children}
```

Comment:
Without the `<br/>` on children there might be inconsistencies on with, I recommend change the defaultProps of children for `<br>`

Reply:
IMO, I don't think the InfoCard should have a `<br />` it would add an extra white space that maybe won't be needed. This component should only just show the info that is passed as props, if in one view there is no need to pass children, we can't add a `<br />` to create consistency in height. For example, in the Dashboard.js, we are using the InfoCard component and I don't need to pass a children, but also, I don't the extra white space that creates the `<br />`. For this case, to create consistency in height, I wrapped every InfoCard in a Col and all InfoCard will have the same height.